indigo/distribution.yaml

```diff
@@ -10158,6 +10159,15 @@ repositories:
       url: https://github.com/ros-drivers/prosilica_gige_sdk.git
       version: hydro-devel
     status: maintained
+  publish_trace:
+    doc:
+      type: git
+      url: https://github.com/wlfws/publish_trace.git
+      version: indigo
```

Comment:
the `version` here should be a valid branch name. Looking at your repository there is a single branch named `master` you likely want to use `master` instead of `indigo` here (Same comment below for the source entry)